app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        if datetime.date.today() > visitor["vaccine"]["expiration_date"]:
```

Comment:
In case you are unsure about the incoming dict structure, it would be preferable to use `.get()` in production settings to retrieve the required values.